devnotes/documentation.md

```diff
@@ -97,11 +107,14 @@ In order to build coreboot image, follow the steps below:
    make menuconfig
    ```
 
-   Navigate to the Mainboard submenu and select the:
-   `Raptor Computign Systems -> Talos II`. Then save the configuration and
-   exit.
+   * Navigate to the **Mainboard** submenu.
+   * As a **Mainboard vendor** select `Raptor Computing Systems`
+   * If it wasn't selected autmatically, as **Mainboard model** select `Talos II`
+   * In the **ROM chip size** option select `512 KB`
+   * As **Size of CBFS filesystem in ROM** set `0x80000`
```

Comment:
```suggestion
   * As **Size of CBFS filesystem in ROM** set `0x80000` (only if this option is
     present)
```
This may not be present if fmap layout is used (e.g. after my SEEPROM implementation)

Reply:
Applied this change